### DISCUSSION TITLE
Problem, cant build before u claim

### DISCUSSION THREAD

**Discussion by Greasyfat** [GriefPrevention/GriefPrevention#2149] [General]
Can someone pls tell me this annoying damn think, i made multiverse flat world, and people are unable to build says entire flat world is claimed.. that u need to claim first before to build anywhere. and this plugin doesnt even have discord -.-

**Comment by bigpresh** [comment]
https://docs.griefprevention.com/configuration/#limiting-which-worlds-griefprevention-runs-on documents the different modes available, you probably want `Survival`?

Also, although a public GH discussion that others can easily find is better than Discord, https://r.griefprevention.com/dumcord/ is linked from the big "Get Help" button on the repo's README.

**Reply by Greasyfat** [reply]
i just want the claim disabled in creative world or just as in normal world, that u can build but if u want save it u need to claim

**Reply by Greasyfat** [reply]
i couldnt find command for it .. why is it even different thre its like that in normal world ehh

**Reply by bigpresh** [reply]
The documentation I linked to explains how to configure it in the config file via the `claims.mode` setting, as far as I recall there's no command to change it.

If you look in your config file, you'll see each world can have its own claims mode setting.
